### ISSUE TITLE
Swap source & target branche selector when creating PR

### ISSUE THREAD

**Issue by tomasandrle** [go-gitea/gitea#23849]
### Feature Description

I think it would be good if the source branch pull down menu was first and the target branch second when creating a PR.

It is confusing when the target branch is first and source second _during creation_ and then reversed once the PR is displayed.

`Select the branch to merge into and the branch to pull from.`

vs

`merge 1 commits from A into B`

**Comment by silverwind** [comment]
How is it on other forges like GitHub or GitLab?

**Comment by tomasandrle** [comment]
GitHub: target left, source right.
GitLab: source left, target right.

I guess this is matter of opinion.

To me the GitLab way seems more intuitive: "what are you merging → into what". Usually I first want to pick the feature branch I just finished working on, then where it should go.

GitLab screenshot:

<img width="1257" alt="Screenshot 2023-03-31 at 21 32 16" src="https://user-images.githubusercontent.com/42672549/229213039-1811b525-cef6-4e11-baa7-ea2c6a5b2807.png">

GitHub:

<img width="1263" alt="Screenshot 2023-03-31 at 21 35 45" src="https://user-images.githubusercontent.com/42672549/229213160-1977d49d-badc-43f0-9af4-f3db06996d9f.png">

**Comment by silverwind** [comment]
Yes, I do tend to agree. `source -> target` seems more intuitive.

**Comment by techknowlogick** [comment]
Another piece of information: Bitbucket does `source -> target` too.

**Comment by wojtekxtx** [comment]
Also agree on this one.
